internal/component/prometheus/exporter/unix/config.go

```diff
@@ -85,6 +85,7 @@ type Arguments struct {
 	Disk        DiskStatsConfig   `alloy:"disk,block,optional"`
 	EthTool     EthToolConfig     `alloy:"ethtool,block,optional"`
 	Filesystem  FilesystemConfig  `alloy:"filesystem,block,optional"`
+	HwMon       HwMonConfig       `alloy:"hwmon,block,optional"`
```

Comment:
you must update the Convert function below to convert this struct to HwMonChipInclude and HwMonChipExclude

Reply:
Thanks added!